tcms/bugs/tests/test_permissions.py

```diff
@@ -58,6 +58,49 @@ def verify_post_with_permission(self):
         self.assertContains(response, 'BUG-%d' % last_bug.pk)
 
 
+class TestEdit(tests.PermissionsTestCase):
+    permission_label = 'bugs.change_bug'
+    http_method_names = ['get', 'post']
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.bug = BugFactory()
+
+        cls.url = reverse('bugs-edit', args=(cls.bug.pk,))
+        cls.post_data = {
+            'summary': 'An edited summary',
+            'reporter': cls.bug.reporter.pk,
+            'assignee': cls.bug.assignee.pk,
+            'product': cls.bug.product.pk,
+            'version': cls.bug.version.pk,
+            'build': cls.bug.build.pk,
+        }
+
+        super().setUpTestData()
+
+    def verify_get_with_permission(self):
+        response = self.client.get(self.url)
+
+        self.assertContains(response, _('Edit bug'))
+        self.assertContains(response, self.url)
+        self.assertContains(response, _(self.bug.summary))
```

Comment:
Summary text comes from the database and it won't be translated. No point in telling Django to translate it.